docs/labels.md

```diff
@@ -0,0 +1,106 @@
+# GitHub Label List
+
+We use GitHub labels to perform issue triage, track and report on development
+progress, plan roadmaps, and automate issue grooming.
+
+To ensure that contributing new issues and PRs remains straight-forward, we would
+like to keep the labels required for submission to a minimum. The remaining
+labels will be added either by automation or manual grooming by other
+contributors and maintainers.
+
+This labels in this list originated within Kubernetes at
+https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md.
+
+## Labels that apply to issues or PRs
+
+| Label | Description | Added By |
+|-------|-------------|----------|
+| api-review                         | Categorizes an issue or PR as actively needing an API review. | Any |
+| area/api                           | Issues or PRs related to an API | Any |
+| area/blog                          | Issues or PRs related to blog entries. | Any |
+| area/conformance                   | Issues or PRs related to conformance testing | Any |
+| area/dependency                    | Issues or PRs related to dependency changes | Any |
+| area/e2e-test                      | Issues or PRs related to end-to-end testing. | Any |
+| area/endpoint/identity             | Issues or PRs related to endpoint identity | Any |
+| area/endpoint/selection            | Issues or PRs related to endpoint selection | Any |
+| area/endpoint/type                 | Issues or PRs related to endpoint type | Any |
+| area/ipam                          | Issues or PRs related to IP address management (IPAM) | Any |
+| area/licensing                     | Issues or PRs related to Antrea licensing | Any |
+| area/monitoring/logging            | Issues or PRs related to logging | Any |
+| area/monitoring/auditing           | Issues or PRs related to auditing | Any |
+| area/monitoring/traffic-analysis   | Issues or PRs related to traffic analysis | Any |
+| area/monitoring/mirroring          | Issues or PRs related to mirroring | Any |
+| area/network-policy/action         | Issues or PRs related to network policy actions | Any |
```

Comment:
The list is really detailed. Would you define high level categories too for users/developers do not know all the details?

Also for bugs, sometimes it is easier to categorize by components like - CNI, Agent, OVS, Controller. What are your thoughts on this?

Reply:
I think we can add component areas as well (these are additive). I agree it is really detailed and I have added essential label suggestions at https://github.com/vmware-tanzu/antrea/pull/347/files#diff-6a3371457528722a734f3c51d9238c13R94-R104. Really, the only required label at submission is the kind and we can ensure that with issue templates. The rest of the labels are for issue triage. We can add the suggested labels (e.g. area/components for kind/bug) in the issue submission templates so that they are immediately available to the contributor during submission. I will research this more and determine the full capability of issue templates.